function Entry()
	assert(game ~= nil)
	assert(Environment ~= nil)
	assert(Lighting ~= nil)
	assert(Players ~= nil)
	assert(ScriptService ~= nil)
	assert(Hidden ~= nil)
	assert(ServerHidden ~= nil)
	assert(PlayerDefaults ~= nil)
	assert(PlayerGUI ~= nil)
	assert(Chat ~= nil)
	assert(Filter ~= nil)
	assert(Camera ~= nil)
	assert(Assets ~= nil)
	assert(Input ~= nil)
	assert(Achievements ~= nil)
	assert(Tween ~= nil)
	assert(CoreUI ~= nil)
	assert(Datastore ~= nil)
	assert(Http ~= nil)
	assert(Insert ~= nil)
	assert(Purchases ~= nil)
	assert(Capture ~= nil)
	assert(Presence ~= nil)
	assert(Preferences ~= nil)
	assert(Insert ~= nil)
end
